### DISCUSSION TITLE
Help is appreciated

### DISCUSSION THREAD

**Discussion by DeathStorm013** [PrismarineJS/mineflayer#2237] [General]
when im running the createbot code its giving me an error for "auth" as unexpected identifier .imma total stranger to java and i tried reading the tutorial but cant figure much ,can someone help me as to wt to do

**Comment by WhoTho** [comment]
Hey

Can you show use the code that you have?

If you are using the "auth" option in the createBot then I assume you are trying to log in with a mircrosoft account. If you are logging in with a mojang account, you do not need the auth line.
```js
const bot = mineflayer.createBot({
  host: "0.0.0.0",
  port: 25565,
  username: "BotUsername",
  password: "BotPassword",
  auth: "microsoft", // do not need if using mojang account
});
```
